projects/igniteui-angular/src/lib/date-range-picker/date-range-picker-inputs.common.ts

```diff
@@ -45,8 +45,9 @@ export class DateRangePickerFormatPipe implements PipeTransform {
     providers: [{ provide: IgxInputGroupBase, useExisting: IgxDateRangeInputsBaseComponent }]
 })
 export class IgxDateRangeInputsBaseComponent extends IgxInputGroupComponent {
+    /** @hidden @internal */
     @ContentChild(NgControl)
-    protected ngControl: NgControl;
+    public ngControl: NgControl;
```

Comment:
Um, why did this become public as well?

Reply:
In `updateValidityOnBlur` if there is `ngControl` we need to get if it is valid or not and update the input directive validity. However, when in two inputs mode `ngModel` could be applied on `date-range-component` or on each of the inputs. In second case we need access to input's `ngModel` from `date-range-component` and it became `public`

Reply:
in the end reverted, not public anymore